.circleci/config.yml

```diff
@@ -0,0 +1,90 @@
+---
+version: 2
+jobs:
+  build:
+    working_directory: ~/pomegranate
+    docker:
+      # The primary container is an instance of the first list image listed. Your build commands run in this container.
+      - image: circleci/ruby:2.3.7-node-browsers
+        environment:
+          RAILS_ENV: test
+          POMEGRANATE_DB: dpul_test
+          POMEGRANATE_DB_HOST: localhost
+          POMEGRANATE_DB_USERNAME: pomegranate
+          POMEGRANATE_DB_PASSWORD: ""
+    steps:
+      - checkout
+      # Restore Cached Dependencies
+      - type: cache-restore
+        name: Restore bundle cache
+        key: pomegranate-{{ checksum "Gemfile.lock" }}
+      # Bundle install dependencies
+      - run: bundle install --path vendor/bundle
+      # Cache Dependencies
+      - type: cache-save
+        name: Store bundle cache
+        key: pomegranate-{{ checksum "Gemfile.lock" }}
+        paths:
+          - vendor/bundle
+      - persist_to_workspace:
+          root: '~/pomegranate'
+          paths: '*'
+  rubocop:
+    working_directory: ~/pomegranate
+    docker:
+      - image: circleci/ruby:2.3.7-node-browsers
+        environment:
+          RAILS_ENV: test
+          POMEGRANATE_DB: dpul_test
+          POMEGRANATE_DB_HOST: localhost
+          POMEGRANATE_DB_USERNAME: pomegranate
+          POMEGRANATE_DB_PASSWORD: ""
+    steps:
+      - attach_workspace:
+          at: '~/pomegranate'
+      # Bundle install dependencies
+      - run: bundle install --path vendor/bundle
+      - run: bundle exec rake rubocop
+  test:
+    working_directory: ~/pomegranate
+    docker:
+      - image: circleci/ruby:2.3.7-node-browsers
+        environment:
+          RAILS_ENV: test
+          POMEGRANATE_DB: dpul_test
+          POMEGRANATE_DB_HOST: localhost
+          POMEGRANATE_DB_USERNAME: pomegranate
+          POMEGRANATE_DB_PASSWORD: ""
+      - image: postgres:10.3
+        environment:
+          POSTGRES_DB: dpul_test
+          POSTGRES_USER: pomegranate
+          POSTGRES_PASSWORD: ""
+    steps:
+      - attach_workspace:
+          at: '~/pomegranate'
+      - run: sudo apt-get update
+      - run: sudo apt-get install libpq-dev libopenjp2-7-dev libmagickwand-dev imagemagick pkg-config
```

Comment:
I don't see any of these being installed in the v1 config, and they aren't mentioned in the README.  Do we want to document additional dependencies that surfaced here?

Reply:
These were installed based upon my findings within other projects.  I had falsely assumed them to be necessary.